addon/components/_private/rdfa-editor/property-editor/modal.gts

```diff
@@ -0,0 +1,65 @@
+import Component from '@glimmer/component';
+import type {
+  ContentTriple,
+  PlainTriple,
+} from '@lblod/ember-rdfa-editor/core/rdfa-processor';
+import OutgoingTripleForm from '../outgoing-triple-form';
+import AuModal from '@appuniversum/ember-appuniversum/components/au-modal';
+import AuButtonGroup from '@appuniversum/ember-appuniversum/components/au-button-group';
+import AuButton from '@appuniversum/ember-appuniversum/components/au-button';
+import WithUniqueId from '../../with-unique-id';
+import { array } from '@ember/helper';
+import { on } from '@ember/modifier';
+import type { SupportedTermType } from '../external-triple-form';
+
+interface Sig {
+  Args: {
+    property?: PlainTriple | ContentTriple;
+    onCancel: () => void;
+    onSave: (property: PlainTriple | ContentTriple) => void;
+    modalOpen: boolean;
+    title?: string;
+    termTypes: SupportedTermType[];
+  };
+}
+
+export default class PropertyEditorModal extends Component<Sig> {
```

Comment:
Looks like the `.hbs` is still there for this one.

Reply:
fixed in [ce3731c](https://github.com/lblod/ember-rdfa-editor/pull/1229/commits/ce3731ca849d2941d4b185e6254781e9fb71b442)